### PR TITLE
Fixing exception when migrating tracker from 0.11 to 0.12/0.13 version

### PR DESCRIPTION
**Pull request by shasha79**
**Proposed changes**:
Fix for https://github.com/RasaHQ/rasa_core/issues/1672

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
Thanks for submitting this PR, please let us know when it's ready to be merged by ticking the relevant box

File: rasa_core/events/__init__.py [hunk 1]
```diff
@@ -229,10 +229,13 @@ def empty():
 
     def as_dict(self):
         d = super(UserUttered, self).as_dict()
+        input_channel = None
```
Comment (tmbo) [inline rasa_core/events/__init__.py]: can you please add a comment that this part is for compatibility reasons?

File: rasa_core/events/__init__.py [hunk 2]
```diff
@@ -759,10 +762,17 @@ def _from_story_string(
 
     def as_dict(self):
         d = super(ActionExecuted, self).as_dict()
+        policy = None
```
Comment (tmbo) [inline rasa_core/events/__init__.py]: same here

**Comment by akelad** [comment]
@shasha79 any chance you could take a look at the suggested changes?

**Comment by tmbo** [comment]
I've added the comments and merged the PR in 28496

**Pull request merged**